lua/damagelogs/client/rdm_manager.lua

```diff
@@ -827,8 +827,9 @@ net.Receive("DL_AllowReport", function()
 
     for i = 1, playerCount do
         local ply = net.ReadEntity()
-        if not IsValid(ply) or not ply:IsPlayer() then continue end
-        dnas[ply] = net.ReadUInt(1) == 1
+        if IsValid(ply) and not ply:IsPlayer() then
```

Comment:
Messed up the logic when removing continue. Should be `and ply:IsPlayer() then` and I'll fix it when I get home from work.

Reply:
To maintain the same logic, this would need to be `if IsValid(ply) and ply:IsPlayer()`.
I don't know if it's necessary to check if the ply object is valid, but `ply:IsPlayer` would fail if it wasn't valid.

Was there a reason why you removed the `continue`?

Reply:
Yea I caught that this morning when I took another look at it. And because Lua doesn't use the continue operator. It threw syntax errors at me.

Reply:
Also IsPlayer() is needed to make sure it isn't an NPC

Reply:
All set, I fixed it in 703adbc